download_mac.sh

```diff
@@ -2,4 +2,10 @@
 
 sudo apt-get install -qq sshpass
 gem install package_cloud
+
+# Download the mac library.
 sshpass -p $SSH_PASS sftp -oStrictHostKeyChecking=no -oPort=2223 -r $SSH_USER@185.19.29.18:/upload down/
```

Comment:
is it possible to download only the interesting file? Because the maven copy will copy all .dylib file. 
Also, for this reason, I'd also put the tag after the `.dylib`. So the format would be traildbjava.dylib-2.0.1